tests/plugins/test_koji_promote.py

```diff
@@ -829,6 +837,46 @@ def test_koji_promote_filesystem_koji_task_id_missing(self, tmpdir, os_env,
         assert 'filesystem_koji_task_id' not in extra
         assert AddFilesystemPlugin.key in caplog.text()
 
+    @pytest.mark.parametrize('additional_tags', [
+        None,
+        ['3.2'],
+    ])
+    def test_koji_promote_image_tags(self, tmpdir, os_env, additional_tags):
```

Comment:
What's `os_env`? I see other tests also take as a parameter so it must be a fixture. It doesn't seem to be used in some of the tests in this file.